Configuration/NodeTypes.yaml

```diff
@@ -94,6 +94,10 @@
 
 'Neos.Neos:Timable':
   properties:
+    '_hiddenInIndex':
+      search:
+        indexing: '${node.hiddenInIndex}'
```

Comment:
this property does not belong to `Neos.Neos:Timable`, but `Neos.Neos:Document`.. see https://github.com/neos/neos-development-collection/blob/master/Neos.Neos/Configuration/NodeTypes.yaml#L180